src/Integration.Vsix/Manifests/VS2017/source.extension.vsixmanifest

```diff
@@ -4,7 +4,8 @@
     <!-- Note: keep the version number in sync with AssemblyInfo.Shared.cs -->
     <Identity Id="SonarLint.36871a7b-4853-481f-bb52-1835a874e81b" Version="5.1.0.0" Language="en-US" Publisher="SonarSource" />
     <DisplayName>SonarLint for Visual Studio 2017</DisplayName>
-    <Description xml:space="preserve">Roslyn based static code analysis: Find and instantly fix nasty bugs and code smells in C#, VB.Net, C, C++ and JS.</Description>
+    <Description xml:space="preserve">SonarLint helps you detect and fix Bugs, Code Smells, and Security Vulnerabilities. 
```

Comment:
Please fix the grammar and I'll commit the updates

Reply:
I think the grammar is ok. For me, the mention of "before the code is committed" is misleading since we treat committed and uncommitted code the same, but it's up to marketing to decide the content.